### DISCUSSION TITLE
Feb 2

### DISCUSSION THREAD

**Discussion by oauch** [GymHubCommunity/GymHub-FE#92] [DailyMeeting]
### ⚽️ 회의 내용

개발 내용 공유
수: 무한 스크롤 개발중
수영: 인강 봤음 😎
슬희: 집으로 복귀
윤혁: 슬라이드

### 🏀 오늘 할 일

수: 무한 스크롤 마무리
수영: 좋아요 (하트)
슬희: 무사히 집으로
윤혁: 2시 이후 정해보기

### 🎱 TMI

수: 매니저님이랑 상담 다 피해갔다.... 닌자 같은 삶
수영: 아침운동 후, 배고파서 쌀국수 곱배기....... 이 남자.... 쌀국수 집에서 회의 진행중
슬희: 비행기에서 집으로 슝
윤혁: 오늘 생일이고, 현재 술이 안 깬다........

**Comment by hanseulhee** [comment]
생일 추카포카 ~~~
